src/other/index.jl

```diff
@@ -201,7 +201,11 @@ end
     end
     idxs[1] isa Symbol && return getindex(x, convert(Vector{Symbol}, idxs))
     throw(ArgumentError("idxs[1] has type $(typeof(idxs[1])); "*
-                        "DataFrame only supports indexing columns with integers, symbols or boolean vectors"))
+                        "DataFrame only supports indexing columns with vectors of integers or symbols"))
```

Comment:
This could be confusing, as we allow indexing data fames with integers, symbols, regexes, and vectors of integers, symbols or Booleans.

Reply:
fixed.